region/maridia/outer.json

```diff
@@ -2988,8 +2994,42 @@
                       "additionalObstacles": ["B"]
                     }
                   ],
+                  "note": "Start the springball jumps from the highest ledge on the wall.",
                   "devNote": "It is possible to bring 2 crabs with double spring ball jump, but is relatively pointless."
                 },
+                {
+                  "name": "Mt. Everest Right Bottom Half Crab Climb with Supers and Jump Assist",
+                  "notable": true,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "canCrabClimb",
+                    {"or": [
+                      "HiJump",
+                      "canSpringBallJumpMidAir"
+                    ]},
+                    "Super",
+                    {"ammo": {
+                      "type": "Super",
+                      "count": 1
+                    }}
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": [],
+                      "additionalObstacles": ["B"]
+                    }
+                  ],
+                  "reusableRoomwideNotable": "Mt. Everest Right Crab Climb with 2 Supers and Jump Assist",
+                  "note": [
+                    "Climbing the right side of Mt. Everest with two supers, ice, and either HiJump or Spring Ball.",
+                    "Requires luring 3 crabs. Either grab a third crab from the morph tunnel to the left or use a third super to grab a different one.",
+                    "Use a super to knock off and freeze a crab midair. Stand on it and let the other crab climb, then jump up and follow it.",
+                    "Be sure two crabs make it to the next section in order to crab climb further.",
+                    "Note: The hardest part of this climb is the pixel precision when jumping around a ledge.",
```

Comment:
This one doesn't have much precise movement.  The hardest part here is freeze timings.